contracts/FastBreak.cdc

```diff
@@ -0,0 +1,682 @@
+/*
+    Fast Break Game Contract
+    Author: Jeremy Ahrens jer.ahrens@dapperlabs.com
+*/
+
+import NonFungibleToken from 0xNFTADDRESS
+import TopShot from 0xTOPSHOTADDRESS
+
+/// Game & Oracle Contract for Fast Break
+///
+pub contract FastBreak: NonFungibleToken {
+
+    /// Contract events
+    ///
+    pub event ContractInitialized()
+    pub event Withdraw(id: UInt64, from: Address?)
+    pub event Deposit(id: UInt64, to: Address?)
+    pub event FastBreakRunCreated(
```

Comment:
I would recommend emitting the UUIDs for all the NFT in these events as well

Reply:
Am emitting this event when the game token is minted. Currently line 349. LMK if this is not what you meant.

emit FastBreakGameTokenMinted(
    id: self.id,
    fastBreakGameID: self.fastBreakGameID,
    serialNumber: self.serialNumber,
    mintingDate: self.mintingDate,
    topShots: self.topShots,
    mintedTo: self.mintedTo
)